gizmo.py

```diff
@@ -0,0 +1,31 @@
+# hello function
+def hello(name, country = "Finland"):
+    print("Hello", name + ", how are things in", country + "?")
```

Comment:
Check out Python's f-string syntax to make this even prettier: https://realpython.com/python-f-strings/